CHANGELOG.md

```diff
@@ -24,6 +24,13 @@ Changelog
 
 * User context is more reliable not cache when the hash mismatches. (E.g. after
   login/logout.)
+  
+* The ContextInvalidationLogoutHandler has been deprecated in favor of the 
+  ContextInvalidationSessionLogoutHandler.  The original handler was called 
```

Comment:
please use backticks around class names (this is markdown, single backtick is good).

double whitespace after the `.` on this line.